Firestore/core/src/firebase/firestore/remote/grpc_root_certificate_finder_apple.mm

```diff
@@ -54,24 +54,24 @@
     if (!bundle) {
       continue;
     }
-
-    NSString* resource = @"gRPCCertificates.bundle/roots";
-    NSString* path = [bundle pathForResource:resource ofType:@"pem"];
-    if (!path) {
-      resource = @"gRPCCertificates-Firestore.bundle/roots";
-      path = [bundle pathForResource:resource ofType:@"pem"];
-    }
-
-    if (path) {
-      LOG_DEBUG("%s.pem found in bundle %s", resource,
-                [bundle bundleIdentifier]);
-      return path;
-    } else {
-      LOG_DEBUG("%s.pem not found in bundle %s", resource,
-                [bundle bundleIdentifier]);
+    // search for the roots.pem file in each of these resource locations
+    NSArray* possibleResources = @[@"gRPCCertificates.bundle/roots",
+                                   @"gRPCCertificates-Firestore.bundle/roots",
+                                   @"roots"];
+      
+    for(NSString* resource in possibleResources){
+      NSString* path = [bundle pathForResource:resource ofType:@"pem"];
+      if(path){
+        LOG_DEBUG("%s.pem found in bundle %s", resource,
+                  [bundle bundleIdentifier]);
+        return path;
+      }
+      else{
```

Comment:
nits: 
- space after `for`, `if`, and `else` 
- no newline before `else` statement 
- space before opening curly brace of `if`, `else`, and `for` bodies

Reply:
Thanks @morganchen12 

I've added the licence blocks but can't run scripts/style.sh cos of broken swiftformat dependencies

Reply:
Feel free to comment out the swiftformat bits if you're not modifying that.